apps/storybook-react/stories/Tooltip.stories.jsx

```diff
@@ -151,3 +153,33 @@ export const WithKnobs = () => {
     </Body>
   )
 }
+
+export function WithDisabledElements() {
+  return (
+    <Body>
+      <TextWrapper>
+        <Typography variant="h3">Tooltip with disabled elements</Typography>
+        <Typography variant="body_long">
+          Firefox, Edge and Chrome supports tooltip on disabled elements.
+        </Typography>
+        <Typography variant="body_long">
+          If you have Safari users, you will need to add inline style to your
+          disabled element, shown in the example below. This will help trigger
+          the mouse events correctly. Unfortunately, this workaround overwrites
+          the &apos;not-allowed&apos; cursor.
+        </Typography>
```

Comment:
Updated the Storybook with a "Disabled element" example, with workaround for Safari browser for the developers to decide what to use.